.github/workflows/actions/manage-doppler-secret/manage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# DOPPLER_TOKEN Must be defined in the ENV
+# The token is obtained from the doppler dashboard for your account
+DOPPLER_PROJECT=$1
+NAMESPACE=$2
+ENVIRONMENT=$3
+DOPPLER_OPERATOR_SECRET_NAME="${NAMESPACE}-${DOPPLER_PROJECT//_/-}"
+DOPPLER_CONFIG="${ENVIRONMENT}_${NAMESPACE}"
+
+function main(){
+  CURRENT_SECRET=$(get_current_secret)
+  if [ "$CURRENT_SECRET" = "SERVICE_TOKEN=" ]; 
+  then
+    echo "[create_doppler_secret] SERVICE_TOKEN is there but it's empty, deleting."
```

Comment:
Inside of github actions, you can append (`>>`)  Markdown into `$GITHUB_STEP_SUMMARY` and it will display that info at the workflow summary level. Useful for stuff you want announced loudly

Reply:
Nice. I'll add a couple key entries for these.